jekyll/_cci2/test-splitting-tutorial.adoc

```diff
@@ -0,0 +1,248 @@
+
+When you set up the project in CircleCI later in this tutorial, you'll select the option to use an existing config.yml that you can edit. The template used with this tutorial is a starter configuration that can be used with Node projects. Read the following section for a quick walkthrough of the configuration, or feel free to skip ahead to set up the project if you're already familiar with setting up Node projects in CircleCI.
+
+=== Configuration walkthrough
+
+* This project uses the configuration `version: 2.1`. This enables CircleCI features such as orbs and resuable configuration. More information on config version 2.1 can be found in the <<configuration-reference#,Configuring CircleCI>> reference document.
+* This project uses the CircleCI Node <<orb-intro#,orb>>. The Node orb is a package of reusable configuration elements that allow you to execute tasks common to Node.js apps and help reduce complexity in your config.yml file. In this particular example, the orb's `install-packages` command is used to easily intall Node packages and configure Yarn as the default package manager using the `pkg-manager` parameter. More details on the Node orb can be found on the https://circleci.com/developer/orbs/orb/circleci/node[Developer Hub].  
+* The project pipeline consists of one workflow called `sample`. This workflow is comprised of one job, named `build-and-test`, which is made up of a few steps to check out the project code, install Node packages and set the default package manager, and run tests.
+
+Test splitting is typically set up within a job. In this tutorial, you will modify the `build-and-test` job to define the number of parallel test runs, where the test suites are located, and how tests should be split (in this case, by timing).
+
+== Step one - Add the project in CircleCI
+
+To get started, you need to get the sample app building as a project on CircleCI. 
+
+. https://github.com/ryanpedersen42/circleci-react-test-splitting/fork[Fork the repository] on GitHub, or download the repository then push the project code to your desired VCS provider.
++
+. Open the https://app.circleci.com[CircleCI web app]. Make sure you're in your personal organization (`\https://app.circleci.com/pipelines/<vcs-provider>/<your-vcs-username>`), then navigate to **Projects**.
++
+. Find the sample app on the list of projects, and click **Set Up Project**.
++
+. You will be prompted to select your config.yml file. In the window, select the "Fast: Take me to a config.yml template that I can edit" option. Click **Set Up Project**.
++
+. You will then see a list of sample configurations. Scroll down within the window until you see **Node (Advanced)**, then click **Select**.
++
+image::{{site.baseurl}}/assets/img/docs/test-splitting-sample-configs.png[Sample config for Node]
++
+. You'll be taken to the configuration. The project actually uses Yarn, so you'll need to edit the following steps so that they use `yarn` instead of `npm`, like so:
```

Comment:
```suggestion
. You will be taken to the configuration. The project actually uses Yarn, so you will need to edit the following steps so that they use `yarn` instead of `npm`, like so:
```